source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,914 @@
+
+      CONTENTURL=http://192.168.25.52
+      VERSIONURL=http://192.168.25.52
 
-.. code-block:: bash
+Example 2: Create a simple mix
+------------------------------
 
-   mixer init --git
+This example shows how to create a simple custom mix using upstream content.
+We'll create an image for a QEMU virtual machine which we can later use to test
+our mix.
 
-.. note::
-   You can use any or all of the above optional flags at the same time, for example:
+We can use the default bundles that were added during intialization, but these
+include the :command:`native-kernel` bundle which has hundreds of drivers we
+don't need for the QEMU virtual machine. So we will modify the default bundle
+set to get a smaller kernel image, which will also be faster to load.
+
+#. Update bundles in mix:
 
    .. code-block:: bash
 
-      mixer init --upstream-version 21060 --mix-version 10 --local-rpms --all-upstream --git
+      mixer bundle remove kernel-native
+      mixer bundle add kernel-kvm
+
+#. Build bundles:
 
-Edit builder.conf
-=================
+   .. code-block:: bash
 
-To configure the mixer tool, edit the :file:`builder.conf` as needed.
+      mixer build bundles
 
-The file :file:`builder.conf` is read automatically from the current
-workspace directory. Use the *--config* flag during initialization
-to specify an alternate path to the file as needed.
+   Look in ~/mixer/update/image/<mix version>/full for the full chrot after the
+   :command:`build` command completes.
 
-The :file:`builder.conf` file has different sections, for example:
+#. Build update content. Browse to your http://localhost site and you'll see
+   the web page is now up, but with no update content. Build the update content:
 
-* The `[Builder]` section provides the mixer tools with the required
-  configuration options. This section defines the path where the generated
-  bundles and update metadata are published.
+   .. code-block:: bash
 
-* The `[swupd]` section contains specific update parameters. The
-  :abbr:`swupd-server (software update server)` creates an update using
-  said specific update parameters.
+      mixer build update
 
-Edit the configuration file according to your needs with the command:
+   Refresh your http://localhost site and now you can see the update content for
+   mix version 10.
 
-.. code-block:: bash
+   Look in ~/mixer/update/www/<mix version> to see the update content in your
+   workspace.
 
-   vim /home/clr/mix/builder.conf
+#. Configure image. Edit the ister configuration file for your image to include
+   all of the bundles you want preinstalled in the image. If this is the first
+   time creating an image, first get a copy of the
+   :file:`release-image-config.json` template file:
 
-Your version of the :file:`builder.conf` file should resemble the
-following example:
+   .. code-block:: bash
 
-.. code-block:: console
+      curl -O https://raw.githubusercontent.com/bryteise/ister/master/release-image-config.json
```

Comment:
Why isn't this being hosted in the clearlinux repo?

Reply:
Because this is not an Intel project. But anyway, ister is being replaced with clear-installer, so this will go away shortly.